tools/cldr-apps/src/main/webapp/js/esm/cldrAjax.js

```diff
@@ -145,6 +145,9 @@ function sendXhr(xhrArgs) {
   if (xhrArgs.postData || xhrArgs.content) {
     options.method = "POST";
     options.data = xhrArgs.postData ? xhrArgs.postData : xhrArgs.content;
+    if (typeof options.data === "object" && xhrArgs.url.includes("/api/")) {
```

Comment:
is this `/api/` check needed given `options.makeJsonPost` ?

Reply:
if `options.makeJsonPost` were set by the caller in cldrAccount then it wouldn't need to be set here, but currently it's only set here in cldrAjax

maybe it would be better for the caller to set `options.contentType` -- anyway this is a hack and there's a lot of room for improvement in making standardizing/simplifying/encapsulating how we set up ajax calls